src/ethereum_test_tools/common/types.py

```diff
@@ -1115,38 +1503,31 @@ def default(self, obj):
                 "gas": hex(obj.gas_limit),
                 "value": hex(obj.value),
                 "input": code_to_hex(obj.data),
-                "to": "0x" + int.to_bytes(obj.to, length=20, byteorder="big").hex()
-                if obj.to is int
-                else obj.to,
+                "to": "0x" + address_to_bytes(obj.to).hex() if obj.to is not None else None,
```

Comment:
I'm assuming that the `address_to_hex(...)` function uses `hex_or_none(...)` within it to return None for the cases where `obj.to` is None here
```suggestion
                "to": address_to_hex(obj.to),
```

Reply:
Fixed :)